README.md

```diff
@@ -1,2 +1,23 @@
-# PluginSkills
-This is attempt in creating a chrome plugin which will maintain my self-development.
+# LifeSkills
```

Comment:
What about name 'LifeGamification' do you want to change it?

Reply:
'LifeGamification' sounds like it is a much bigger project. For version 0.1 'LifeSkills' sounds much better to me. We could change the name with later versions.